wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/clients/basisdaten/WahlenClientImpl.java

```diff
@@ -0,0 +1,43 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorstandservice.clients.basisdaten;
+
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.configuration.Profiles;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.eai.aou.client.WahldatenControllerApi;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.eai.aou.model.WahlDTO;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand.KonfigurierterWahltagModel;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand.WahlModel;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand.WahlenClient;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.WlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import java.util.List;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+@Profile(Profiles.NOT + Profiles.DUMMY_CLIENTS)
+public class WahlenClientImpl implements WahlenClient {
+
+    private final ExceptionFactory exceptionFactory;
+    private final WahldatenControllerApi wahldatenControllerApi;
+    private final WahlenClientMapper wahlenClientMapper;
+
+    @Override
+    public List<WahlModel> getWahlen(final KonfigurierterWahltagModel wahltag) throws WlsException {
+        final Set<WahlDTO> wahlDTOs;
+        try {
+            wahlDTOs = wahldatenControllerApi.loadWahlen(wahltag.wahltag(), wahltag.nummer());
+        } catch (final Exception exception) {
+            log.info("exception on loadwahl from external", exception);
+            throw exceptionFactory.createTechnischeWlsException(ExceptionConstants.KOMMUNIKATIONSFEHLER_MIT_AOUEAI);
```

Comment:
```suggestion
            throw exceptionFactory.createTechnischeWlsException(ExceptionConstants.KOMMUNIKATIONSFEHLER_MIT_BASISDATEN);
```

Reply:
siehe commit [84bbcc5](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/84bbcc5f9988a587d205eaae7b75ac976313df36)